check-oracle.sh

```diff
@@ -0,0 +1,184 @@
+#!/bin/bash
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# used for oracle ORAchk.  install, run and uninstall ORAchk
+#
+# Check if we're using the Mac stock getopt and fail if true
+# shellcheck disable=SC2034
+out="$(getopt -T)"
+if [ $? != 4 ]; then
+    echo -e "Your getopt does not support long parameters, possibly you're on a Mac, if so please install gnu-getopt with brew"
+    echo -e "\thttps://brewformulas.org/Gnu-getopt"
+    exit 1
+fi
+
+# set to 'echo ' to only dislay commands
+DEBUG_CMD='echo '
+DEBUG_CMD=''
+
+ORA_SWLIB_BUCKET_PARAM='^gs://.+[^/]$'
+
+ORACLE_SERVER=''
+AHF_DIR='AHF'
+AHF_FILE=''
```

Comment:
As with oracle_server, if we're using ora-swlib-bucket and ahf-file only to generate a GCS path to pull the AHF zipfile, I'd prefer just to accept a simple --ahf-location=gs://bucket/file/location/AHF.zip parameter, and validate that it starts with gs:// and that it exists.